src/renderers/webgpu/utils/WebGPUTextureUtils.js

```diff
@@ -202,25 +202,36 @@ class WebGPUTextureUtils {
 
 		const { width, height, depth, levels } = options;
 
+		let format;
+
 		if ( texture.isFramebufferTexture ) {
 
 			if ( options.renderTarget ) {
 
-				options.format = this.backend.utils.getCurrentColorFormat( options.renderTarget );
```

Comment:
Would it be possible to keep updating the `options` object as before? 

`options` is not only used in `createTexture()` but also in subsequent code like `updateTexture()`. To avoid side effects, it would be beneficial if `options` does not loose any information with these changes.

Reply:
Could you also temporarily update the builds? I can then build a rawgit URL for WebXR testing.

Reply:
@Mugen87 Interestingly, the AA problem is not related to this PR but in WebGPU / WebGPUBackend. I haven't 
discovered the origin yet, if you have any suggestions? I went back a few releases to test and the problem remains.

| WebGPUBackend | WebGLBackend |
| -- | -- |
| <img width="306" height="310" alt="image" src="https://github.com/user-attachments/assets/42d30a0e-d794-4b75-8912-527dfac56bd5" /> | <img width="316" height="300" alt="image" src="https://github.com/user-attachments/assets/ee99295b-a92c-4c7b-9a8b-ee04f8d589a0" /> |
| https://jsfiddle.net/8oxjL7y1/1/ | https://jsfiddle.net/8oxjL7y1/2/ |

Reply:
Um, is this issue related to flat shading? When comparing without AA the WebGL and WebGPU backend, I can see rendering differences at the edges. `WebGLRenderer` matches with the WebGL backend, btw.

The MSAA is just emphasizing the issue in WebGPU.